data/fields/parking.json

```diff
@@ -11,7 +11,8 @@
             "carports": "Carports",
             "garage_boxes": "Garage Boxes",
             "rooftop": "Rooftop",
-            "sheds": "Sheds"
+            "sheds": "Sheds",
+            "street_side": "Street-side"
```

Comment:
```suggestion
            "street_side": "Streetside"
```

Reply:
See my comments regarding the spelling of street-side above.